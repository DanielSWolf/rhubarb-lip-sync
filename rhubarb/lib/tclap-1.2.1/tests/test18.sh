#!/bin/sh

# failure
../examples/test4 -Bs --Bs asdf > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test18.out; then
	exit 0
else
	exit 1
fi

