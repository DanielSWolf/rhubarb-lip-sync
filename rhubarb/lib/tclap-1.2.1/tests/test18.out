my failure message: 
-s -- Couldn't find match for argument
